src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -231,6 +231,7 @@ void CoilCoolingDX::instantiateFromInputSpec(EnergyPlusData &state, const CoilCo
         ShowSevereItemNotFound(state, eoh, "Availability Schedule Name", input_data.availability_schedule_name);
         errorsFound = true;
     }
+    this->performance->coilCoolingDXAvailSched = this->availSched;
```

Comment:
The availability schedule specified in the parent object is passed to the `CoilCoolingDXCurveFitPerformance` object. This schedule pointer is then passed to the schedule pointer variable in the `CoilCoolingDXCurveFitOperatingMode` object.